### PR TITLE
[5.0] Update Dispatcher Docblocks

### PR DESCRIPTION
**Pull request by reshadman**

### CODE REVIEW

File: src/Illuminate/Bus/Dispatcher.php [hunk 1]
```diff
@@ -73,6 +73,7 @@ public function dispatchFromArray($command, array $array)
 	 *
 	 * @param  mixed  $command
 	 * @param  \ArrayAccess  $array
+	 * @param  mixed $extras
```
Comment (GrahamCampbell) [inline src/Illuminate/Bus/Dispatcher.php]: mixed is not correct

File: src/Illuminate/Bus/Dispatcher.php [hunk 2]
```diff
@@ -73,9 +73,10 @@ public function dispatchFromArray($command, array $array)
 	 *
 	 * @param  mixed  $command
 	 * @param  \ArrayAccess  $array
+	 * @param  array $extras
```
Comment (taylorotwell) [inline src/Illuminate/Bus/Dispatcher.php]: Needs to be two spaces after `array` intead of just one.

**Comment by GrahamCampbell** [comment]
The contract also needs updating.

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

**Pull request merged**